src/js/base/helpers.cy.js

```diff
@@ -11,6 +11,7 @@ context('Handlebars helpers', function() {
       template: hbs`
         <div class="test-null">{{matchText "Patient Name" null}}</div>
         <div class="test-match">{{matchText "Patient Name" "Patient"}}</div>
+        <div class="test-match-substring">{{matchText "Testpatient Name" "patient" includeSubstrings=true}}</div>
```

Comment:
Doing `{{matchText "TestPatient Name" "Patient" includeSubstrings=true}}` does not work interestingly. Having the  capitalized `Patient`.